app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -172,8 +172,8 @@ Survey getOne(boolean exceptionIfEmpty) {
             if (notDeleted) {
                 scan.addFilterCondition(DELETED_PROPERTY, equalsNumber("0"));
             }
-            // Scans will not sort as queries do. Sort Manually.
-            List<DynamoSurvey> surveys = Lists.newArrayList(surveyMapper.scan(DynamoSurvey.class, scan));
+            List<DynamoSurvey> surveys = Lists.newArrayList(surveyMapper.scanPage(DynamoSurvey.class, scan).getResults());
```

Comment:
From the documentation, scanPage only returns the first page of results. This code path is used for the "getAll" APIs, which (from the name) are intended to get all. So we really should use scan() instead of scanPage().